app/assets/javascripts/discourse/app/components/user-menu/menu.js

```diff
@@ -66,6 +68,28 @@ const CORE_TOP_TABS = [
       return !this.currentUser.likes_notifications_disabled;
     }
   },
+
+  class extends UserMenuTab {
+    get id() {
+      return REVIEW_QUEUE_TAB_ID;
+    }
+
+    get icon() {
+      return "flag";
+    }
+
+    get panelComponent() {
+      return "user-menu/reviewables-list";
+    }
+
+    get shouldDisplay() {
+      return this.currentUser.can_review;
+    }
+
+    get count() {
+      return this.currentUser.get("reviewable_count");
```

Comment:
```suggestion
      return this.currentUser.reviewable_count;
```

Reply:
I'm using `get` intentionally here so that Ember tracks the `reviewable_count` property and re-renders the tab when it changes (there's a test for this that fails if `get` is removed).

Reply:
:+1: